src/components/pages/HomePage/index.tsx

```diff
@@ -182,8 +178,7 @@ export default function HomePage() {
           </List>
         </Box>
         <Typography fontSize={24} py={4} textAlign='center'>
-          If you have questions or would like free trial access, email Moshe
-          Siegel at siegel.moshes@gmail.com
+          To contact us, email Moshe Siegel at siegel.moshes@gmail.com
```

Comment:
If you don't mind the blue text, we could make the email a link:

`<Typography fontSize={24} py={4} textAlign='center'>
          To contact us, email Moshe Siegel at <Link href={'mailto:siegel.moshes@gmail.com'}>siegel.moshes@gmail.com</Link>
</Typography>`